ui/src/views/network/VpnDetails.vue

```diff
@@ -130,13 +130,11 @@ export default {
               duration: 0
             })
             this.fetchData()
-            this.parentFetchData()
```

Comment:
Might need a refresh after enabling / disabling the VPN

Reply:
@davidjumani same https://github.com/apache/cloudstack/pull/4782#discussion_r670206261